Examples/Io/CMakeLists.txt

```diff
@@ -6,3 +6,4 @@ add_subdirectory(NuclearInteractions)
 add_subdirectory(Obj)
 add_subdirectory(Performance)
 add_subdirectory(Root)
+add_subdirectory_if(Svg ACTS_BUILD_PLUGIN_ACTSVG)
```

Comment:
```suggestion
add_subdirectory_if(Svg ACTS_BUILD_PLUGIN_ACTSVG)
```

Reply:
Can't see the difference?

Reply:
I see it's a newline change, for Cmake files we can leave this I'd say